src/engine/graphic/GRStaffManager.cpp

```diff
@@ -122,6 +122,9 @@ GRStaffManager::GRStaffManager(GRMusic * p_grmusic, ARPageFormat * inPageFormat,
 		// Apply default layout settings
 		GuidoGetDefaultLayoutSettings (&this->settings);
 	}
+    
```

Comment:
why copying the map ? the staff manager already refers to the GRMusic, create an API in GRMusic and use it instead of creating redundancies